### DISCUSSION TITLE
Navigating the E-Signature Landscape: DocuSign’s Potential Sale and OpenSign’s Role

### DISCUSSION THREAD

**Discussion by andrew-opensignlabs** [OpenSignLabs/OpenSign#281] [Q&A]
Hello OpenSignites,

We’re at a crossroads in the e-signature landscape, and it’s an exciting time for us to discuss the future and the role OpenSign™ can play. Recent news suggests that DocuSign, one of the major players in the e-signature space, is exploring options to sell their business. This development could have significant implications for their customers and the industry as a whole.

Why This Matters:

	1.	Change in Service Dynamics: The sale of a major service provider like DocuSign often leads to changes in pricing, customer service quality, and the overall user experience – a phenomenon commonly referred to as “enshittification.”
	2.	Opportunity for Alternatives: Such transitions open up opportunities for users to seek more stable, reliable, and user-centric alternatives.
	3.	Open Source Advantage: As an open-source platform, OpenSign™ offers a unique value proposition. We’re not just a service provider; we’re a community-driven solution that evolves with the needs and contributions of our users.

Discussion Points:

	•	Experiences with DocuSign: Share your experiences or concerns with DocuSign, especially regarding the potential sale.
	•	Needs and Expectations: What do you look for in an e-signature solution? How can OpenSign™ better meet these needs?
	•	Community Contributions: Ideas and suggestions for enhancing OpenSign™ as a robust alternative.
	•	Future Developments: Discuss what features or improvements you’d like to see in OpenSign™.

This is more than just a discussion about a competitor’s potential sale; it’s a conversation about the future of e-signature technology and how we, as a community, can shape it to better serve users like you. Your insights, experiences, and suggestions are invaluable to this journey. Let’s explore together how OpenSign™ can redefine what it means to be an effective, user-friendly, and community-driven e-signature solution.

Looking forward to an engaging and insightful discussion!

**Comment by moblesmarti** [comment]
Hola, yo acabo de llegar a DocuSign (todavía no he conseguido hacerlo funcionar:) después de estar revisando, en mi opinión, varios mal llamados “Open Source Alternative”.
La etiqueta Open Source está de moda y es muy fácil colocar una parte del código en Github y las características importantes dejarlas como Enterprise. Es una opción legítima pero engañosa.
Si das, quieres recibir y para invertir tiempo en colaborar en una aplicación Open Source debe ser una aplicación abierta de verdad.
Hay aplicaciones que han conseguido (con merecimiento) una gran comunidad y un Enterprise que funciona. Nextcloud es un gran ejemplo de aplicación con todas sus características abiertas y unos servicios Enterprise paralelos.
La venta, hoy por hoy, de una aplicación Open Source suele ser a grupos inversores y a estos grupos no les importa lo mas mínimo la comunidad, su único interés está en repartir beneficios.
Saludos
